pkg/cmd/pulumi/package_gen_sdk.go

```diff
@@ -50,6 +52,13 @@ func newGenSdkCommand() *cobra.Command {
 			if err != nil {
 				return err
 			}
+			if version != "" {
+				pkgVersion, err := semver.Parse(version)
+				if err != nil {
+					return fmt.Errorf("invalid version %q: %w", version, err)
+				}
+				pkg.Version = &pkgVersion
```

Comment:
Should we write out what the schema version we're overwriting _was_ just for user information?

Reply:
Done ✅